offlineimap/imapserver.py

```diff
@@ -198,7 +198,7 @@ def __md5handler(self, response):
         challenge = response.strip()
         self.ui.debug('imap', '__md5handler: got challenge %s'% challenge)
 
-        passwd = self.__getpassword()
+        passwd = self.__getpassword().encode('ascii')
```

Comment:
Hmm, thinking about this a bit more, I think we should force users to use unicode in their password (and username) and we forcefully encode to UTF-8 for both py2 and py3.

* py2: `password = u"password_with_e_acute_é"
* py3: `password =   "password_with_e_acute_é"